ansible_collections/arista/avd/roles/eos_validate_state/ANTA-Preview.md

```diff
@@ -151,6 +151,9 @@ title: Ansible Collection Role eos_validate_state - Preview Integration with ANT
 - (New) AvdTestAPIHttpsSSL (No Ansible tags, use the new `skipped_tests` variable instead)
   - VerifyAPIHttpsSSL: Validate eAPI HTTPS SSL profile status.
 
+- (New) AvdTestStun (No Ansible tags, use the new `skipped_tests` variable instead)
+  - VerifyStunClient: Validate the STUN client is configured with the specified IPv4 source address and port exclusively for WAN scenarios. Additionally, validate the public IP and port if provided. Please add the test in a custom ANTA catalog to validate public IP and port.
```

Comment:
1 lets remove the part where we suggest custom ANTA catatlog for public IP and port. AVD does not offer this capability to be derived from the structured config

2. update using the suggestion I made in the test docstring. What the test is really checking is that there is an entry in the STUN client translations for the given local interfaces in each path-group.